lib/data/landing_page_content_items/be_kinder.yaml

```diff
@@ -5,13 +5,8 @@ blocks:
     href: /landing-page/goals
   - text: Tasks
     href: /landing-page/tasks
-    children:
-      - text: Child 1
-        href: /a
-      - text: Child 2
-        href: /b
```

Comment:
Haven't done a full code review as I assume this is more of a backend PR, but are we able to keep the child links in so that the pages match the designs on mobile?  

The key was renamed to `items` as part of https://github.com/alphagov/frontend/pull/4285, so if you change the key to `items` you should see the child links on mobile.

(I've also renamed `children` to `items` in https://github.com/alphagov/frontend/pull/4312)

Thanks!

Reply:
Restored.

Reply:
Thanks!